lib/widget/switch.dart

```diff
@@ -102,12 +115,13 @@ class SwitchState extends FormFieldWidgetState<EnsembleSwitch> {
 
     // wraps around FormField to get all the form effects
     return InputWrapper(
-      type: EnsembleSwitch.type,
+      // type: EnsembleTripleStateSwitch.type,
+      type: '',
```

Comment:
put the type in. This is to help with exceptions.

Reply:
Done